legal-api/src/legal_api/resources/business/business_tasks.py

```diff
@@ -38,6 +41,7 @@ class TaskListResource(Resource):
 
     @staticmethod
     @cors.crossdomain(origin='*')
+    @jwt.requires_auth
     def get(identifier):
```

Comment:
I think there's an old ticket describing that this (or "some") API methods do not check the JWT. Since this is now implemented (:clap), that ticket can possibly be closed.

Reply:
@severinbeauvais different issue